src/d3d12/d3d12_renderer.cpp

```diff
@@ -246,17 +234,52 @@ namespace wr
 
 	void D3D12RenderSystem::Render_MeshNode(MeshNode* node)
 	{
-		for (auto& mesh : node->m_model->m_meshes)
-		{
-			auto n_mesh = static_cast<D3D12Mesh*>(mesh);
-			auto frame_idx = GetFrameIdx();
+	}
+
+	//Render batches
+	void D3D12RenderSystem::RenderMeshBatches(SceneGraph& scene_graph)
+	{
+		
+		auto &batches = scene_graph.GetBatches();
```

Comment:
I don't believe the style guide mentions this but please put the reference or pointer symbols to the left or centered. Everywhere else its either left or centered.